docs/spec.md

```diff
@@ -414,7 +415,8 @@ Set one or more parameters. Only supported if the server previously declared tha
   "parameters": [
     { "name": "/int_param", "value": 3 },
     { "name": "/float_param", "value": 4.1 },
-    { "name": "/byte_array_param", "value": "QUJDRA==", "type": "byte_array" }
+    { "name": "/byte_array_param", "value": "QUJDRA==", "type": "byte_array" },
+    { "name": "/float_array_param", "value": [1.1, 2, 3.3], "type": "float64_array" }
```

Comment:
Should both `float64` and `float64_array` lines be in  both examples?
Unclear to me why not have them in both fi the byte_array is in both

Reply:
agreed, will do